lib/manageiq/graphql/types/query.rb

```diff
@@ -13,25 +13,27 @@ module Types
           argument :id, types.ID
 
           resolve -> (obj, args, ctx) {
-            ::Service.find(args[:id])
+            service = ::Service.find(args[:id])
+            ::Rbac.filtered_object(service)
           }
         end
 
         field :services, !types[Service], "List available services" do
           argument :tags, types[types.String]
 
           resolve -> (obj, args, ctx) {
-            if args[:tags]
-              ::Service.find_tagged_with(:all => args[:tags].join(" "), :ns => Classification::DEFAULT_NAMESPACE)
-            else
-              ::Service.all
-            end
+            services = if args[:tags]
+                         ::Service.find_tagged_with(:all => args[:tags].join(" "), :ns => Classification::DEFAULT_NAMESPACE)
```

Comment:
So, with this, dos this return a scope, or an array?  If it is an array, can we get it to return a scope possibly?

Reply:
@NickLaMuro  TBH I have no idea. I stole this directly from how tags are looked up in the REST API.

Reply:
@NickLaMuro Looks like it does indeed return a relation 🎉